test/Libraries/DynamoPythonTests/PythonEvalTests.cs

```diff
@@ -165,7 +165,7 @@ public void CPythonEngineIncludesTraceBack()
             catch (Exception exc)
             {
                 // Trace back is not available for this call, but we still get a reasonable message back
-                Assert.AreEqual(@"SyntaxError : ('invalid syntax', ('<string>', 3, 13, ""print 'hello'\n""))", exc.Message);
+                Assert.AreEqual(@"SyntaxError : ('invalid syntax', ('<string>', 3, 7, ""print 'hello'\n""))", exc.Message);
```

Comment:
why did this change?

Reply:
That's a good question. That's the type of error where the traceback is built-in. My guess is that was a result of the change in the version of Python, from 3.7 to 3.8, but I couldn't find it in the release notes. The column number changed to point to the beginning of the section where the invalid syntax was detected, so technically this seems more correct.

Reply:
Maybe this https://bugs.python.org/issue38535?